extendr-engine/README.md

```diff
@@ -1,4 +1,14 @@
-# extendr-engine
+# `extendr-engine`
 
-This crate is mostly use for testing `extendr-api`. See
-`extendr-api` for more details.
+This crate is mostly use for testing [`extendr-api`]. See
+[`extendr-api`] for more details.
+
+[`extendr-api`]: https://extendr.github.io/extendr/extendr_api/index.html
+
+Provides a singleton instance of the R interpreter.
+
+Only call this from `main()` if you want to run stand-alone.
+
+Its principal use is for testing.
```

Comment:
This line is a bit redundant.

Reply:
I think we should change this description in another PR.
I didn't write this myself :(
I just need a little more time with the codebase.. 
Is it okay I mark this resolved?

Reply:
Oh okay.